### PR TITLE
Register Spark UDFs

### PR DESCRIPTION
**Pull request by ljfgem**
### What changes are proposed in this pull request, and why are they necessary?

Register several Spark UDFs.
### How was this patch tested?

Tested on target views

### CODE REVIEW

**Comment by 7AXEL** [comment]
Check out this project: https://github.com/7AXEL/Web2APK

**Review by dihu-linkedin — APPROVED** [review]
lgtm

File: coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java [hunk 1]
```diff
@@ -673,6 +673,13 @@ public boolean isOptional(int i) {
     createAddUserDefinedFunction("com.linkedin.policy.decoration.udfs.RedactSecondarySchemaFieldIf", ARG1, family(
         SqlTypeFamily.BOOLEAN, SqlTypeFamily.ANY, SqlTypeFamily.ARRAY, SqlTypeFamily.CHARACTER, SqlTypeFamily.ANY));
 
+    createAddUserDefinedFunction("com.linkedin.groot.runtime.udf.spark.HasMemberConsentUDF", ReturnTypes.BOOLEAN,
```
[the remark this answers is not in the document]
Comment (ljfgem) [reply coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java:676]: `StaticHiveFunctionRegistry` doesn't contain Hive UDFs only (check the java doc of this class), it contains all the UDFs (i.e. Hive UDF, Transport UDF, etc)

**Review by 1fanwang — APPROVED** [review]

**Review by KevinGe00 — APPROVED** [review]
lgtm

**Pull request merged**